File: src/main/java/com/p2p/lending/dao/CertifrecordDao.java
package com.p2p.lending.dao;

import java.util.List;
import java.util.Map;

import com.p2p.lending.pojo.Certifrecord;

public interface CertifrecordDao extends BaseDao<Object, Object> {

	public List<Certifrecord> queryCertifrecord(Map<String, Object> map);

	public int updateCertifrecord(Map<String, Object> map);

	public int addCertifrecord(Map<String, Object> map);

}

File: src/main/java/com/p2p/lending/dao/CertifrecordDao.kt
package com.p2p.lending.dao

import com.p2p.lending.entity.Certifrecord

interface CertifrecordDao : BaseDao<Any, Any> {

    fun queryCertifrecord(map: Map<String, Any>): List<Certifrecord>

    fun updateCertifrecord(map: Map<String, Any>): Int

    fun addCertifrecord(map: Map<String, Any>): Int

}